python-sdk/tests/benchmark/Makefile

```diff
@@ -25,29 +25,33 @@ clean:
 	@rm -f ../../airflow.db
 
 # Takes approximately 7min
-setup_gke:
+setup_gke: clean
 	@cd infrastructure/terraform && \
 		terraform init && \
 		terraform apply
 
-container:
+run: clean
+	@AIRFLOW__ASTRO_SDK__SNOWFLAKE_STORAGE_INTEGRATION_GOOGLE=gcs_int_python_sdk GCP_BUCKET=dag-authoring ./run.sh
+
+container: clean
 	@echo "Building and pushing container $(CONTAINER_REGISTRY)"
-	@gcloud auth configure-docker
+	#@gcloud auth configure-docker
```

Comment:
Do we not need this anymore?

Reply:
I'll uncomment and improve this, @kaxil , by replacing it with
```
@gcloud auth configure-docker gcr.io
```

By default, this command adds lots of items to `credHelpers` in `~/.docker/config.json` - which slows down docker building. It even prompts the following:

```
WARNING: A long list of credential helpers may cause delays running 'docker build'. We recommend passing the registry name to configure only the registry you are using.
After update, the following will be written to your Docker config file located at [/Users/al-cht01/.docker/config.json]:
 {
  "credHelpers": {
    "gcr.io": "gcloud",
    "us.gcr.io": "gcloud",
    "eu.gcr.io": "gcloud",
    "asia.gcr.io": "gcloud",
    "staging-k8s.gcr.io": "gcloud",
    "marketplace.gcr.io": "gcloud"
  }
}

Do you want to continue (Y/n)?
```